decidim-meetings/app/views/decidim/meetings/admin/registrations/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title">
+      <%= title %>
+      <span class="exports dropdown tiny button button--simple button--title" data-toggle="export-dropdown"><%= t "actions.export", scope: "decidim.admin" %></span>
+      <div class="dropdown-pane" id="export-dropdown" data-dropdown data-auto-focus="true" data-close-on-click="true">
+        <ul class="vertical menu add-features">
+          <% %w{csv json}.each do |format| %>
```

Comment:
Or maybe (sorry) we should have a way to register exporters, so that external plugins can add their own ones (maybe this is too big for this PR?)